oka/chap2/knock10.py

```diff
@@ -0,0 +1,9 @@
+import subprocess
+
+
+path = "C:\\Users\\16t215\\Desktop\\hightemp.txt"
+with open(path, encoding="utf-8") as f:
```

Comment:
ファイルを開くモードは出来るだけ指定しよう

| char | mode |
----|----
| w | 書き込み |
| r | 読み込み |
| a | 書き込み（追記） |


```suggestion
with open(path, "r", encoding="utf-8") as f:
```